src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,310 @@
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationNote.name',
+                defaultMessage: 'Authorization Note',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        startSingleDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.startSingleDate.fullName',
+                defaultMessage: 'Use of Collections start/single date',
+              },
+              name: {
+                id: 'field.uoc_common.startSingleDate.name',
+                defaultMessage: 'Start/single date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        endDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              fullName: {
+                id: 'field.uoc_common.endDate.fullName',
+                defaultMessage: 'Use of Collections end date',
+              },
+              name: {
+                id: 'field.uoc_common.endDate.name',
+                defaultMessage: 'End date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        userGroupList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          userGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.uoc_common.userGroup.name',
+                  defaultMessage: 'User',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: CompoundInput,
+                props: {
+                  tabular: true,
+                },
+              },
+            },
+            user: {
+              [config]: {
+                messages: defineMessages({
+                  name: {
+                    id: 'field.uoc_common.user.name',
+                    defaultMessage: 'Name',
+                  },
+                  fullName: {
+                    id: 'field.uoc_common.user.fullName',
+                    defaultMessage: 'User name',
+                  },
+                }),
+                view: {
+                  type: AutocompleteInput,
+                  props: {
+                    source: 'person/local,person/shared,organization/local,organization/shared',
+                  },
+                },
+              },
+            },
+            userType: {
+              [config]: {
+                messages: defineMessages({
+                  name: {
+                    id: 'field.uoc_common.userType.name',
+                    defaultMessage: 'User Type',
```

Comment:
Sentence case. Also make this the fullName, and add a name that is just "Type". Otherwise you'll have a table labeled "User", and a column labeled "User type", which is redundant.